azure-pipelines.yaml

```diff
@@ -112,12 +112,12 @@ steps:
       targetType: filePath
       filePath: ./script/create_package.sh
 
-  - task: Bash@3
-    displayName: Start Kakfa in single node with Java and Python
-    inputs:
-      targetType: filePath
-      filePath: ./test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/server/start-kafka-test-environment.sh
-      workingDirectory: test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/server
+  # - task: Bash@3
```

Comment:
is there any reason for removing this stage in this pr ??

Reply:
These older tests require us to create docker images with the older versions of the function apps. Building all the apps together is causing the memory on the pipeline to run out [error link](https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=71139&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=dee4e6ec-be1b-5671-4678-ab293b4004c5&l=42906).

Reply:
As discussed offline, we can revert the change for now